zargparse.py

```diff
@@ -52,6 +53,8 @@ def arg_string(self) -> str:
             # This is needed so that completion works for subcommands when a
             # global flag is used.
             result += ':'
+        if self.choices:
+            result += self.options[0]+":("+' '.join(self.choices)+")"
```

Comment:
Thanks for implementing this! One small fix is needed here since "choices" can be something other than a string (e.g. an int). Also, I think it's a little more readable if you use the format method like so. This should work:

`            result += "{} :({})".format(self.options[0], ' '.join([str(c) for c in self.choices]))
`

I tested this by modifying the example script like this:
```
diff --git a/examples/future b/examples/future
index 55152de..ca627f6 100755
--- a/examples/future
+++ b/examples/future
@@ -26,7 +26,7 @@ if __name__ == '__main__':
     # Travel subcommand
     parser_travel = subparsers.add_parser('travel', help='Travel in time')
     parser_travel.add_argument('--year', '-y', type=int, default=1955,
-                               help='Year to travel to.')
+                               help='Year to travel to.', choices=(1955, 2015, 1885))
```

Reply:
Good catch!